src/rules/no-export.js

```diff
@@ -0,0 +1,49 @@
+import { getDocsUrl, isTestCase } from './util';
+
+let exportNodes = [];
+let hasTestCase = false;
+const messageId = 'unexpectedExport';
+export default {
+  meta: {
+    docs: {
+      url: getDocsUrl(__filename),
+    },
+    messages: {
+      [messageId]: `Do not export from a test file.`,
```

Comment:
No need to use a variable for `messageId`. I find it easier to read as a literal and we'll type check it when migrating to TS

Reply:
done